pulsar-proxy/src/main/java/org/apache/pulsar/proxy/server/ProxyConfiguration.java

```diff
@@ -111,6 +111,9 @@
     )
     private String brokerWebServiceURLTLS;
 
+    @FieldContext(category = CATEGORY_BROKER_DISCOVERY, doc = "")
+    private int maxMessagesSize = 5233640;
```

Comment:
Proxy should not have this config option. Just repeat what broker had.

Reply:
Yeah. will fix